ancar/portfolio/src/main/webapp/gallery.html

```diff
@@ -2,18 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Anca's Image Gallery</title>
+    <title>Sasha's Image Gallery</title>
```

Comment:
who is sasha?

Reply:
my cat, it's a gallery of pictures of her. Should I add a comment there?

Reply:
:) ok I was thinking it was by mistake.. ;)